pythoncalculator.py

```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y

```

Comment:
I think this needs to be changed to 
```suggestion
    return x*y

```